### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by sub-22**
pull init project

### CODE REVIEW

**Comment by sub-22** [comment]
ready

**Comment by sub-22** [comment]
ready

**Comment by ngocvt-0484** [comment]
ready

**Comment by ngocvt-0484** [comment]
ready

**Comment by ngocvt-0484** [comment]
commented

**Comment by sub-22** [comment]
Screenshot from 2021-06-16 08-25-56

**Comment by sub-22** [comment]
ready

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -0,0 +1 @@
+e/6WQ45TFg2EDsXnCTMSqZrpy+NOWg8/XYTUO6tB8w3P8iZwSanOVeQMPTaupbIH9HFT5pxYj8RAnLDE3u9uIQjh2JYFwAPyLnT8WzUCv4c2cMMx2gSkxAhJJsRwDBNnQJfabHoBwgMEzvZIT/CstSoZU9t3EUkwFhK1sf8oMR9xul8BI2WZYPR4wVzl5G0cEBngpVKVIcUCD0E6TqBugnILwi9o5RWXD1zLGtD8gumyCHfgkafdReV9FJHoq5cwY0bcXrVWK1ITn3717AZ9Zhkt+p67sckQivGxvSd7MAtyoDwBs6koKc5Bx2sLOTPPwJsQXq5wJeRWDTfekV8kHiWOYaNFHEzofr9vS14fbHyp1WbIzHfZqLQKSlMqhnj5Fid9lFWnlsiTSivkMbOSXWDv/YdbRoWp+csG--8I5XNgR0JkGaBPYa--JGd+MmgA4hz8FDH2H70z7Q==
```
Comment (ghost) [inline config/credentials.yml.enc]: ignore file này em ơi

**Review by ghost — COMMENTED** [review]
cmt

**Pull request merged**